src/main/java/io/strimzi/kafka/bridge/http/converter/HttpJsonMessageConverter.java

```diff
@@ -30,6 +30,17 @@
 
 public class HttpJsonMessageConverter implements MessageConverter<String, byte[], Buffer, Buffer> {
 
+    private Integer partition;
+
+    public HttpJsonMessageConverter(int partition) {
+        this.partition = partition;
+    }
+
+    public HttpJsonMessageConverter() {
+        super();
+        this.partition = null;
+    }
```

Comment:
as said partition-id on the URL is not part of this PR, please remove all the stuff related to it.